src/api/environment.cc

```diff
@@ -209,6 +209,8 @@ void SetIsolateCreateParamsForNode(Isolate::CreateParams* params) {
     // heap based on the actual physical memory.
     params->constraints.ConfigureDefaults(total_memory, 0);
   }
+  params->embedder_wrapper_object_index = 0;
+  params->embedder_wrapper_type_index = 1;
```

Comment:
I'm not entirely sure what's supposed to be stored at the type index /cc @MayaLekova

Reply:
I think they can be both zero. V8 currently doesn't uses the type index. The object index is the internal field in the object that stores the receiver, the first argument to fast callback (`FastNumber()` and `FastBigint()` in this PR.)

Reply:
The type index was supposed to provide the internal field where the type info for custom embedder types is stored. The type index will most probably be unnecessary and we'll remove it from the API. This is currently an open question in the design, but the object index might as well become unnecessary, if we decide to move the object unwrapping, currently done during optimization, to the embedder callback (at runtime).

Reply:
that would be interesting because we don't really need the receiver here, maybe the fast apis could just accept `uint32_t*`/`uint64_t*` as the first argument when that's supported.

Reply:
How would you perform type checks then? Just to make sure I understand your point correctly, the functions currently ported to the fast API don't need the receiver, right? I still see the `receiver` parameter being used, but is it only to workaround the lack of support for `uint64_t`?

Reply:
@MayaLekova i meant something like this, not sure if it would actually be possible:

```cc
void FastBigInt(uint64_t* values) {
  values[0] = uv_hrtime();
}

void SlowBigInt(const FunctionCallbackInfo<Value>& args) {
  FastBigInt(args[0]->As<ArrayBuffer>()->GetBackingStore()->Data());
}
```

Reply:
Thanks for the example! This seems to me like a valid use case and it *should* be possible to create such, at least for the already supported types, e.g. with `uint32_t`. Although due to the current design, you might need to define `WrapperTraits<uint32_t>` in this case (to use a pointer to it), which seems silly. I'll think if we can simplify this use case.